src/azure-openapi-validator/tests/resources/DescriptionMustNotBeNodeName.json

```diff
@@ -0,0 +1,116 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Microsoft Azure Redis Cache Management API",
+        "description": "Some cool documentation.",
+        "version": "2014-04-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "basePath": "/",
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}": {
+            "delete": {
+                "tags": [
+                    "Redis"
+                ],
+                "operationId": "Redis_Delete",
+                "description": "Deletes a Redis cache.",
+                "x-ms-long-running-operation": true,
+                "parameters": [
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the resource group."
+                    },
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "name.",
```

Comment:
is this the one violating the rule, if so, should we be more explicit and call it "propertyNameSameAsDescription" or something like that?